index.html

```diff
@@ -1 +1 @@
-<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0; url=0.14/index.html"><link rel="canonical" href="http://ramdajs.com/0.14/index.html" /></head><body></body></html>
\ No newline at end of file
+<!DOCTYPE html><html><head><link rel="canonical" href="http://ramdajs.com/0.14/index.html" /><script>window.location = "0.14/index.html" + location.hash;</script></head><body></body></html>
```

Comment:
strange -- i added that and another little tweak, then `git add lib/doc/redir.tpl.js; git commit --amend; git push -f origin redir:redir` but it hasn't changed